packages/nimble-components/src/table/models/table-layout-manager.ts

```diff
@@ -56,14 +57,19 @@ export class TableLayoutManager<TData extends TableRecord> {
 
     /**
      * Sets up state related to interactively sizing a column.
+     * @param divider The divider element that was clicked on
+     * @param pointerId The pointerId of the pointer that started the drag
      * @param dragStart The x-position from which a column size was started
-     * @param activeColumnDivider The divider that was clicked on
+     * @param activeColumnDivider The 1-based index of the divider that was clicked on
      */
     public beginColumnInteractiveSize(
+        divider: HTMLElement,
```

Comment:
nit: can we call this input `activeColumnDividerElement` to keep a consistent name through (also better contrasts to the `activeColumnDivider`)